frontend/front/src/pages/Public/Pages/Home/HeroPage.js

```diff
@@ -48,74 +56,101 @@ const HeroPage = ({
-                      href={link}
-                      target="_blank"
-                      rel="noopener"
-                      underline="always"
-                      sx={{
-                        color: "var(--color-text-9)",
-                        fontWeight: 800,
-                        textDecoration: "solid underline 3px",
-                        textUnderlinePosition: "under",
-                        textDecorationColor: "var(--color-text-6)",
-                      }}
-                    >
-                      {textBold}
-                    </MuiLink>
-                  )}
-                  {text2}
-                </Typography>
-              </Box>
-
-              <Button
-                component={RouterLink}
-                to="/public/benefits-heat-pump"
-                onClick={() => window.scrollTo(0, 0)}
+      <Box className="text-overlay">
+        <Box
+          sx={{
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "center",
+            alignItems: "center",
+            height: "100%",
+          }}
+        >
+          <Box
+            sx={{
+              mt: 2,
+              flexGrow: 1,
+              display: "flex",
+              flexDirection: "column",
+              justifyContent: "center",
+              alignItems: "center",
+            }}
+          >
+            <AnimatedBox triggerOnce={false}>
+              <TitleHero
+                text={title.toUpperCase()}
+                titleBold={titleBold.toUpperCase()}
+              />
+              <Typography
+                variant="h6"
                 sx={{
-                  width: "auto",
-                  minWidth: "200px",
-                  maxWidth: "250px",
-                  height: "50px",
-                  color: "var(--color-text-2)",
-                  background: "var(--bgColor-3)",
-                  borderRadius: "50px",
-                  px: 3,
-                  "&:hover": {
-                    backgroundColor: "var(--bgColor-3)",
-                  },
+                  color: "var(--color-text-9)",
+                  px: { xs: 1, md: 20 },
+                  textAlign: "center",
                 }}
               >
-                Benefits of heat pumps
-              </Button>
-            </Box>
-          </AnimatedBox>
+                {text1}
+                {link !== "" && (
+                  <MuiLink
+                    href={link}
+                    target="_blank"
+                    rel="noopener"
+                    underline="always"
+                    sx={{
+                      color: "var(--color-text-9)",
+                      fontWeight: 800,
+                      textDecoration: "solid underline 3px",
+                      textUnderlinePosition: "under",
+                      textDecorationColor: "var(--color-text-6)",
+                    }}
+                  >
+                    {textBold}
+                  </MuiLink>
```

Comment:
There is a missing space between this and the preceding text in Creole:
![image](https://github.com/codeforboston/urban-league-heat-pump-accelerator/assets/93659375/9d160819-070b-4615-9116-c44629c5eb24)

Reply:
Fixed it: https://github.com/codeforboston/urban-league-heat-pump-accelerator/pull/489/commits/c35bcf4bebe6c99998f37ce8cc83a793ace9bdee